composer.lock

```diff
@@ -4115,16 +4115,16 @@
         },
         {
             "name": "oat-sa/extension-tao-itemqti",
-            "version": "v29.21.5",
+            "version": "v29.22.0",
```

Comment:
Version is ok, cause it respected the last one from May release, so no backporting required :) 
https://github.com/oat-sa/extension-tao-itemqti/releases/tag/v29.21.5